src/asset.rs

```diff
@@ -61,16 +61,17 @@ impl Asset {
         self.commission_year + self.process.parameter.lifetime
     }
 
-    /// Get the activity limits for this asset in a particular time slice
-    pub fn get_activity_limits(&self, time_slice: &TimeSliceID) -> RangeInclusive<f64> {
-        let limits = self.process.activity_limits.get(time_slice).unwrap();
+    /// Get the energy limits for this asset in a particular time slice
+    /// This is an absolute max and min on the PAC energy produced/consumed in that time slice.
```

Comment:
I think you usually have a blank line for longer comments:

```suggestion
    /// Get the energy limits for this asset in a particular time slice.
    ///
    /// This is an absolute max and min on the PAC energy produced/consumed in that time slice.
```